### PR TITLE
Add message/redpanda digi

### PR DESCRIPTION
**Pull request by vakumar1**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
Add a Redpanda broker digi that allows consumers/producers to read/write from/to "topics." The broker endpoint is hardcoded and was initially created as a utility digi in the anysource dspace.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: message/redpanda/deploy/values.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+group: message.digi.dev
+image: vakumar23/redpanda.v1.message.digi.dev:latest
```
Comment (JamsheedMistri) [inline message/redpanda/deploy/values.yaml:2]: Was this built with `digi build -a` for cross-platform compatibility??

File: message/redpanda/driver/handler.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import sys
+import subprocess
+
+import digi
+
+# external endpoint for redpanda broker (should be anysource host + nodeport)
+ADVERTISED_BROKERS = "dspace.anysource.io:30292"
```
Comment (JamsheedMistri) [inline message/redpanda/driver/handler.py:7]: Why is this hardcoded?

File: message/redpanda/driver/handler.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+if __name__ == '__main__':
+    try:
+        subprocess.check_call(f"rpk redpanda start --advertise-kafka-addr '{ADVERTISED_BROKERS}' &", shell=True)
+    except subprocess.CalledProcessError:
+        digi.logger.info("unable to start rpk")
```
Comment (JamsheedMistri) [inline message/redpanda/driver/handler.py:13]: Is the digi called `redpanda` or `rpk`? IMO it should be consistent for readability. If it's `redpanda`, then this log message should read `"unable to start redpanda"`.

File: message/redpanda/driver/handler.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+    digi.logger.info("started rpk")
```
Comment (JamsheedMistri) [inline message/redpanda/driver/handler.py:15]: Same as above

**Review by JamsheedMistri — CHANGES_REQUESTED** [review]
Are you planning on running this as a digi space? If so, additional modifications are needed in `/Makefile`, `/cmd/digi/digi/digi.go`, `/cmd/digi/space/command.go`, and `/model/Makefile`. See my headscale PR [here](https://github.com/digi-project/digi/pull/46/files) for reference.

File: message/redpanda/driver/handler.py (hunk 2 above)
Comment (silveryfu) [reply message/redpanda/driver/handler.py:7]: We should perhaps make this configurable via the redpanda's model

Comment (silveryfu) [reply message/redpanda/driver/handler.py:7]: @vakumar1 shall we fix this (add to the model) and then we can merge?

**Review by JamsheedMistri — APPROVED** [review]
LGTM!

**Pull request merged**